lib/game/round.ex

```diff
@@ -0,0 +1,39 @@
+defmodule Reactor.Game.Round do
+  use GenServer
+
+  @colors ["red", "green", "yellow", "blue", "orange", "pink"]
+
+  def start_link(game_id, users) do
+    GenServer.start_link(__MODULE__, [game_id, users])
+  end
+
+  def submit(_hello, _world) do
+    {:ok}
+
+    # update_state
+    # check state
+    # if everyone submitted
+    # emit event
+  end
+
+  ##Server Callbacks
+
+  def init([game_id, users]) do
+    #get random colors
+    #set it in state
+    #emit to game_event_manager
+    colors = Enum.take_random(@colors, 4)
```

Comment:
Generally it's good practice not to do work in the `init` method – the reason being is starting a new process is sync so nothing else can be started via the supervisor until this has completed. Much better to `Process.send(self, :msg)`